core/tests/protractor_utils/forms.js

```diff
@@ -185,23 +190,22 @@ var RichTextEditor = async function(elem) {
   var closeRteComponentButtonLocator = by.css(
     '.protractor-test-close-rich-text-component-editor');
   // Set focus in the RTE.
-  await waitFor.elementToBeClickable(rteElements.first());
-  await (await rteElements.first()).click();
+  var rteElementFirst = await rteElements.first();
+  await action.click('rte Element First', rteElementFirst);
 
   var _appendContentText = async function(text) {
-    await (await rteElements.first()).sendKeys(text);
+    await action.sendKeys('First rte Element ', rteElementFirst, text);
```

Comment:
Please capitalize 'RTE' in the first argument (for consistency)

Reply:
done !